contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -201,10 +212,22 @@ func (c *devCfg) RegisterFlags(fs *flag.FlagSet) {
 	)
 }
 
+func (c *devCfg) validateConfigFlags() error {
+	if (c.balancesFile != "" || c.txsFile != "") && c.genesisFile != "" {
+		return errors.New("cannot specify `balances-file` or `txs-file` along with `genesis-file`")
```

Comment:
Please consider making this error global

Reply:
done a39d826ba5b397cba2d896da3c436fc25090aeec